goinsta.go

```diff
@@ -205,6 +206,32 @@ func Export(inst *Instagram, writer io.Writer) error {
 	return err
 }
 
+// ExportString exports selected *Instagram object as base64 encoded string
+func ExportString(inst *Instagram, writer io.Writer) (string, error) {
+	url, err := neturl.Parse(goInstaAPIUrl)
+	if err != nil {
+		return "", err
+	}
+
+	config := ConfigFile{
+		ID:        inst.Account.ID,
+		User:      inst.user,
+		DeviceID:  inst.dID,
+		UUID:      inst.uuid,
+		RankToken: inst.rankToken,
+		Token:     inst.token,
+		PhoneID:   inst.pid,
+		Cookies:   inst.c.Jar.Cookies(url),
+	}
+	bytes, err := json.Marshal(config)
```

Comment:
I think it's better to use `Export` and `ImportReader` instead of writing.

Reply:
sry, what do you mean exactly? ^^

Reply:
I do not think we should add this feature to our library. It depends on the developer that will use GoInsta.

Reply:
So you mean by default export a string instead of writing it to a file directly, and the let the user decide if he wants to save it as a file?

Reply:
BTW here is what I mean,

```go
func saveToBase64(insta *goinsta.Instagram) (string, error) {
	buffer := &bytes.Buffer{}
	err := goinsta.Export(insta, buffer)
	if err != nil {
		return "", err
	}
	return base64.StdEncoding.EncodeToString(buffer.Bytes()), nil
}
```

```go
func readFromBase64(base64EncodedString string) (*goinsta.Instagram, error) {
	base64Bytes, err := base64.StdEncoding.DecodeString(base64EncodedString)
	if err != nil {
		return nil, err
	}
	return goinsta.ImportReader(bytes.NewReader(base64Bytes))
}

```